scripts/test.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
+
+thread_count=$(getconf _NPROCESSORS_ONLN)
+
 if which flutter >/dev/null; then
     flutter pub global activate junitreport
-    flutter test --coverage
+    flutter test --concurrency=$thread_count --coverage
     TEST_CODE=$?
     
     # coverage
     flutter pub global activate remove_from_coverage
     flutter pub global run remove_from_coverage:remove_from_coverage -f coverage/lcov.info -r '\.g\.dart$'
 else
-    dart test --coverage=coverage
-    dart test --file-reporter='json:TEST-report.json'
+    dart test --concurrency=$thread_count --coverage=coverage
+    dart test --concurrency=$thread_count --file-reporter='json:TEST-report.json'
```

Comment:
Do we run the tests twice here???

Reply:
hm the tests do indeed run twice :D /2 the time again owo